shared/chat/inbox/row/new-conversation.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import React, {PureComponent} from 'react'
+import {Text, Icon} from '../../../common-adapters'
+import {globalStyles, globalColors, globalMargins} from '../../../styles'
+
+class NewConversation extends PureComponent<{}> {
+  render() {
+    return (
+      <div
+        style={{
+          ...globalStyles.flexBoxRow,
+          alignItems: 'center',
+          backgroundColor: globalColors.blue,
+          flexShrink: 0,
+          minHeight: 48,
```

Comment:
the other conversations have `height: 56` as do the designs

Reply:
ah, that part was just moved from existing, can change